security/crowdsec/src/opnsense/scripts/OPNsense/CrowdSec/reconfigure.py

```diff
@@ -69,6 +69,7 @@ def configure_bouncer(settings):
     config['log_dir'] = '/var/log/crowdsec'
     config['blacklists_ipv4'] = 'crowdsec_blacklists'
     config['blacklists_ipv6'] = 'crowdsec6_blacklists'
+    config['retry_initial_connect'] = True
```

Comment:
```suggestion
    config['retry_initial_connect'] = true
```

PSR12 wants these as lowercase

Reply:
Ah lol this is a Python file 😂

Reply:
Correct ... if you use `true` in Python, yaml_dump will generate a string constant `'true'` in the output. I found out by trial and error.

Reply:
Hehe yeah sorry for the noise.